doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -0,0 +1,76 @@
+.. _bluetooth_mesh_brg:
+
+Subnet Bridge
+#############
+
+The Bluetooth Mesh Subnet Bridge feature allows a node to relay messages between different subnets in a Mesh network.
+
+The Subnet Bridge feature includes two models:
+
+.. toctree::
+   :maxdepth: 1
+
+   brg_cfg_srv
+   brg_cfg_cli
+
+The Bridge Configuration Server model is mandatory for supporting the Subnet Bridge feature.
+The Bridge Configuration Client model is optional and allows nodes to configure the Subnet Bridge on other nodes.
+These models define the necessary states, messages, and behaviors for configuring and managing the Subnet Bridge feature.
+
+Concepts
+********
+
+The Subnet Bridge feature introduces several concepts to enable message relaying between different subnets in a Mesh network.
+
+Subnet
+======
+
+A subnet is a group of nodes within a Mesh network that share a common network key, enabling them to communicate securely at the network layer.
+Each subnet operates independently, with nodes exchanging messages exclusively within that group.
+A node may belong to multiple subnets at the same time.
+
+Subnet Bridge
+=============
+
+A Subnet Bridge is a node in a Mesh network that connects multiple subnets, allowing communication between them by relaying messages across subnet groups.
+Only nodes with the Subnet Bridge functionality enabled can perform this role.
+The Subnet Bridge node has a primary subnet, based on the primary NetKey, which handles the IV Update procedure and propagates updates to other subnets.
+The secondary subnets, on which messages are relayed, are referred to as *bridged subnets*.
+The Bridge Configuration Client can get the Subnet Bridge feature status of a node by sending a **Bridge Configuration Get** message to the Bridge Configuration Server model on the target node, using the :c:func:`bt_mesh_brg_cfg_cli_subnet_bridge_get` function.
+The Client can also request to get the list of subnets that are currently bridged by a node by sending a **Bridged Subnets Get** message by calling the :c:func:`bt_mesh_brg_cfg_cli_bridged_subnets_get` function.
```

Comment:
You will probably want to add some breathing room as this is rendering as literally just one big paragraph (unless you insert blank lines), which is hard to visually "parse".
This comment applies to other blocks of text in your PR but this one seems to be the one where readability is impacted the most

Also, .rst files should wrap at 100 characters like all other files in Zephyr.